cfme/common/provider.py

```diff
@@ -51,13 +60,11 @@ class BaseProvider(Taggable, Updateable, SummaryMixin, Navigatable):
 
     @classmethod
     def add_base_type(cls, nclass):
-        cls.base_types[nclass.category] = nclass
-        return nclass
+        pass
```

Comment:
I was going to remove them completely - no one should really be using them anyway, the pass was just to get things working I should actually have removed them.